src/components/Projects.tsx

```diff
@@ -1,7 +1,214 @@
 import React from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+import {
+  space,
+  SpaceProps,
+  layout,
+  LayoutProps,
+  background,
+  BackgroundProps,
+  FlexboxProps,
+  flexbox,
+  PositionProps,
+  position,
+  ColorProps,
+  color,
+  TypographyProps,
+  typography,
+  border,
+  BorderProps,
+} from "styled-system";
+import stairs from "./assets/project-page/stairs.png";
+import shell from "./assets/project-page/shell.png";
+import eye from "./assets/project-page/eye.png";
+import statue from "./assets/project-page/statue.png";
+import dragon from "./assets/project-page/dragon.png";
+import knife from "./assets/project-page/knife.png";
+import mask from "./assets/project-page/mask.png";
+import spider from "./assets/project-page/spider.png";
+import magnify from "./assets/project-page/magnify.png";
+
+type BgProps = BackgroundProps &
+  ColorProps &
+  LayoutProps &
+  FlexboxProps &
+  TypographyProps &
+  LayoutProps &
+  SpaceProps;
+
+const Background = styled.div<BgProps>`
+  ${background};
+  ${color};
+  ${layout};
+  ${flexbox};
+  ${space};
+`;
+
+type LogoProps = SpaceProps &
+  LayoutProps &
+  FlexboxProps &
+  PositionProps &
+  ColorProps &
+  SpaceProps &
+  TypographyProps &
+  BorderProps &
+  ColorProps;
+
+const HomeLink = styled.a<LogoProps>`
+  ${space};
+`;
+
+const HomeLogo = styled.p<LogoProps>`
+  ${flexbox};
+  ${typography}
+  ${color}
+`;
+
+const BuyButton = styled.button<LogoProps>`
+  ${space};
+  ${layout};
+  ${color};
+  ${typography};
+  ${flexbox};
+  ${border}
+  ${position};
+`;
+
+const Icon = styled.a<LogoProps>`
+  ${layout};
+  ${position};
+  ${flexbox};
+  ${space};
+  ${position};
+  a:hover {
+    width: 2em;
+  }
+`;
 
 const Projects = () => {
-  return <div>Projects</div>;
+  return (
+    <Background
+      backgroundSize="cover"
+      backgroundImage="url('./assets/home-page/background.jpg')"
+      backgroundPosition="center"
+      backgroundRepeat="no-repeat"
+      bg="black"
+      height="100vh"
+      display="flex"
+      flexDirection="column"
+      alignItems="center"
+      mx="auto"
+    >
+      <HomeLink p={30}>
+        <Link to="/" style={{ textDecoration: "none" }}>
```

Comment:
We can add `text-decoration: none` to some global styles somewhere? index.css?

Reply:
Fo sho'